repairnator/scripts/config/repairnator.cfg

```diff
@@ -27,6 +27,7 @@ SCANNER_NB_HOURS=1 # Number of hours to inspect to get last builds (e.g. 4 will
 
 NB_THREADS=4 # Number of concurrent docker container to run
 DAY_TIMEOUT=1 # Global timeout to stop the docker execution
+SKIP_DELETE=1 # Skip the deletion of docker containers
```

Comment:
IMHO the option should be placed at the end of the file under `### Switches` category. And its default value should be 0. Indicate that 1 would mean to keep the docker containers instead of deleting it.

Reply:
Fixed.